omnibus/files/private-chef-cookbooks/private-chef/libraries/private_chef.rb

```diff
@@ -555,10 +555,12 @@ def warn_if_cred_mismatch(opts)
 WARN2
         end
       elsif pass_in_config
-        warning = <<WARN
+        unless PrivateChef["insecure_addon_compat"]
+          warning = <<WARN
 #{config_key_desc} is configured in plain in chef-server.rb -- remove
```

Comment:
should this say "plaintext" rather than "plain"?  Existing bug..but while we are here.